### PR TITLE
BBS

### PR DESCRIPTION
**Pull request by mtsmfm**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -27,5 +27,7 @@ class Application < Rails::Application
       g.helper false
       g.assets false
     end
+
+    config.app_generators.scaffold_controller = :scaffold_controller
```
Comment (tricknotes) [inline config/application.rb]: Move to `config.app_generators` block?

File: config/routes.rb [hunk 2]
```diff
@@ -18,5 +18,8 @@
     concerns :site
   end
 
+  resources :topics
```
Comment (tricknotes) [inline config/routes.rb]: Move to inside of `concerns :site` ?

**Comment by tricknotes** [comment]
I left some comments however looks great :ok_woman:

File: config/routes.rb (hunk 2 above)
Comment (mtsmfm) [inline config/routes.rb]: ouch! :dizzy_face:

File: config/application.rb (hunk 1 above)
Comment (mtsmfm) [inline config/application.rb]: :lipstick:

**Comment by tricknotes** [comment]
This is awesome :sparkles:

**Pull request merged**